### DISCUSSION TITLE
See status in the future

### DISCUSSION THREAD

**Discussion by R3ffoTz** [dala318/nordpool_planner#17] [Q&A] [answered]
Is it possible to see future on/off status change on the sensor?

**Comment by dala318** [comment, marked as answer]
Yes, it should be in the additional attributes of the binary_sensor
![image](https://github.com/user-attachments/assets/91c484dc-01a0-4811-8222-a0410a9d9f34)
You should be able to extract that info via a template sensor, some inspiration [here](https://community.home-assistant.io/t/how-to-create-a-template-sensor-that-reads-an-attribute-from-a-sensor/437374) 

Onwards I'm hoping to get some more time to further enhance the integration to provide som more sensors like as you mention "start_at(_time)" etc. but at the moment this is what available.

**Comment by dala318** [comment]
Created #34  to track the request
